backend/tests/unit/dal/Entities/AccessRequestAgencyTest.cs

```diff
@@ -0,0 +1,73 @@
+using FluentAssertions;
+using Pims.Core.Test;
+using Pims.Dal.Entities;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Xunit;
+
+namespace Pims.Dal.Test.Entities
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
+    [Trait("category", "entities")]
+    [Trait("group", "entity")]
+    [ExcludeFromCodeCoverage]
+    public class AccessRequestAgencyTest
+    {
+        #region Tests
+        [Fact]
+        public void AccessRequestAgency_Id_Constructor()
+        {
+            // Arrange
+            var accessRequest = EntityHelper.CreateAccessRequest(5);
```

Comment:
I'm generally nervous around using the same id for two different values in a test like this. What if we had some code that accidentally assigned an agency id to an access request id or vice versa? Nice to have unique ids for each object to prevent that.

Reply:
Technically it is meaningless what the ids are.  The only thing important is that the code applies the id of the appropriate object.  Humanly speaking though I think it makes sense to use different ids.  I'll update.